### PR TITLE
UICIRC-374: Move Save button to the footer

### PR DESCRIPTION
**Pull request by annamelnyk**
### Description
Move `Save` button to the footer at **Settings > Circulation > Other Settings | Loan history**

---

### Issues
https://issues.folio.org/browse/UICIRC-374
https://issues.folio.org/browse/UICIRC-375

 --- 

### Screenshots
image
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## [1.13.0] (IN PROGRESS)
 
+* Settings > Circulation > Loan history | Move Save button to the footer. Refs UICIRC-375.
+* Settings > Circulation > Other Settings | Move Save button to the footer. Refs UICIRC-374.
```
Comment (VictorSoroka1) [inline CHANGELOG.md]: Please add new entries at the bottom of the list.

File: src/settings/LoanHistory/LoanHistoryForm.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.loanHistoryForm {
+  height: 100%;
+}
```
Comment (VictorSoroka1) [inline src/settings/LoanHistory/LoanHistoryForm.css]: Please add an empty line at the end of the file.

File: src/settings/CheckoutSettingsForm.js [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
+  Pane,
+  Row,
+  Select,
+  TextField,
+  Label,
+  PaneFooter,
+} from '@folio/stripes/components';
 import stripesForm from '@folio/stripes/form';
-import { Field, FieldArray, getFormValues } from 'redux-form';
+import {
```
Comment (VictorSoroka1) [inline src/settings/CheckoutSettingsForm.js]: Please move third party imports to the fourth line after react-intl, so third-party imports are grouped together. Also, add an empty line after them, so folio dependencies are divided from them.

Overall please the stick to the following format:
```
import React from 'react';
... other third-party imports

import stripesForm from '@folio/stripes/form';
... other folio imports

import { patronIdentifierTypes } from '../constants';
... other project relative imports

import CSS from './CheckoutSettingsForm.css';
... other CSS related imports
```

File: src/settings/CheckoutSettingsForm.js [hunk 4]
```diff
@@ -36,34 +54,44 @@ class CheckoutSettingsForm extends React.Component {
...
-      >
-        <FormattedMessage id="ui-circulation.settings.checkout.save" />
-      </Button>
+      <PaneFooter
+        renderEnd={(
+          <Button
+            id="clickable-savescanid"
+            type="submit"
+            buttonStyle="primary paneHeaderNewButton"
+            disabled={(pristine || submitting)}
```
Comment (SergiyVSergiyenko) [inline src/settings/CheckoutSettingsForm.js]:
```suggestion
            disabled={pristine || submitting}
```
I would get rid of parentheses here.

**Pull request merged**